package.json

```
@@ -12,21 +12,22 @@
     "url": "https://github.com/sissbruecker/linkding-extension/issues"
   },
   "homepage": "https://github.com/sissbruecker/linkding-extension#readme",
+  "type": "module",
   "scripts": {
+    "dev": "rollup -c -w",
     "build": "rollup -c",
-    "dev": "rollup -c -w"
+    "generate": "./build.sh"
```

```suggestion
    "package": "./build.sh"
```

Might be more indicative of what it does. Can also rename the shell script if we agree on this.

---

I like it, will rename both. 👍🏼